portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,15 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Erica Smith's Portfolio</h1>
+      <p>My name is Erica Smith. I'm from Ashland, VA and I'm a rising sophomore at NC A&T SU.
+         This summer, I am a STEP intern with Google.  
```

Comment:
I'm not sure if the newline here is for readability or for display purposed.  If it's the former, feel free to ignore this comment.  It it's the latter, HTML ignores newlines and you need to put this in a separate paragraph (<p>) or add a break (<br>)